### PR TITLE
vscode-abl issue #66

### PR DESCRIPTION
**Pull request by gquerret**
Squashed a few commits, and rebased on top of master

### CODE REVIEW

**Comment by chriscamicas** [comment]
thank you @gquerret
does this PR replace the other one #19 ?

**Comment by gquerret** [comment]
Yes ! I forgot to close the other one.

**Comment by gquerret** [comment]
@chriscamicas Do you need anything else to merge the code ? Otherwise I can do that myself.

**Comment by gquerret** [comment]
@chriscamicas Don't merge, I've seen a few issues, I'll add comments for @PeterJudgeZA

File: abl.tmLanguage.json [hunk 1]
```diff
@@ -532,18 +1566,43 @@
...
-      "match": "(?i)(?<=^|\\s)(true|false|yes|no)(?!\\w|-)",
-      "name": "constant.language.source.abl"
+      "match": "(?i)(?<=^|\\s*)(true|false|yes|no|\\?)(?!\\w|-)",
+      "name": "constant.language.abl"
+    },
+    "punctuation-colon": {
+      "name": "punctuation.terminator.abl",
+      "match": ":"
+    },
+    "punctuation-colon": {
```
Comment (gquerret) [inline abl.tmLanguage.json:1603]: Duplicate object key.

**Comment by gquerret** [comment]
I will reopen this afternoon.

**Pull request closed without merging**